### ISSUE TITLE
Failed to extract lib-dynload/_sha3.cpython-39-x86_64-linux-gnu.so: decompression resulted in return code -1!

### ISSUE THREAD

**Issue by theChaosCoder** [borgbackup/borg#6419]
## Have you checked borgbackup docs, FAQ, and open Github issues?

yes

## Is this a BUG / ISSUE report or a QUESTION?

Issue

#### Your borg version (borg -V).
borg 1.2.0

#### Operating system (distribution) and version.
Qnap nas 64bit
Linux qnap 5.10.60-qnap
Intel(R) Celeron(R) J4025 CPU @ 2.00GHz

I downloaded https://github.com/borgbackup/borg/releases/download/1.2.0/borg-linux64
But I get this strange message when I try to run borg (same with -V)
`Failed to extract lib-dynload/_sha3.cpython-39-x86_64-linux-gnu.so: decompression resulted in return code -1!`

Has some experienced a similar issue?

**Comment by ThomasWaldmann** [comment]
Can you try the `borg-linuxold64` binary? Sometimes the Linux on such devices is rather old...

**Comment by theChaosCoder** [comment]
Tried the "old" version: same result

**Comment by ThomasWaldmann** [comment]
Is your TEMP full while running borg (it tries to extract itself to temp location)?

**Comment by theChaosCoder** [comment]
Hmm yes tmp seems to be very small (64M). Available 62.5M

I tried this now

```
export TMPDIR=/share/homes/myuser/borgtmp
./borg-linux64
Error loading Python lib '/tmp/_MEI5Zn46j/libpython3.9.so.1.0': dlopen: /lib/libc.so.6: version `GLIBC_2.28' not found (required by /tmp/_MEI5Zn46j/libpython3.9.so.1.0)
```

**Comment by ThomasWaldmann** [comment]
OK, so it was just 1st your temp dir running full and 2nd your glibc being too old.
